packages/web_benchmarks/test/src/browser_test_json_samples.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert' show jsonDecode;
+
+// JSON Event samples taken from running an instrumented version of the
+// integration tests of this package that dumped all the data as captured.
+
+/// To test isBeginFrame.
+final Map<String, dynamic> beginMainFrameJson = jsonDecode('''
```

Comment:
It's probably worth encoding the version information into these names, since the `name` field is version-specific.

Reply:
Yes, added a `_89plus` suffix to the variable names (and some Dart-docs) to understand that they've been sampled from Chromium 89+.

(If the format changes again, we can do `_XXplus` for future version XX of chrome, and if we want to add some captured JSON from the old version, I'd do `_pre89` or similar.)